src/OmniSharp.Roslyn/OmniSharpTextLoader.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace OmniSharp
+{
+    public class OmniSharpTextLoader : TextLoader
```

Comment:
we can remove this code now as well https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Script/ScriptProjectSystem.cs#L236
I can do that in a separate PR though, no need to hold up here